cmd/operator-opamp-bridge/metrics/reporter.go

```diff
@@ -122,16 +121,17 @@ func NewMetricReporter(
 
 	// Create some metrics that will be reported according to OpenTelemetry semantic
 	// conventions for process metrics (conventions are TBD for now).
-	reporter.processCpuTime, err = reporter.meter.Float64ObservableGauge(
+	reporter.processCpuTime, err = reporter.meter.Float64ObservableCounter(
 		"process.cpu.time",
-		instrument.WithFloat64Callback(reporter.processCpuTimeFunc),
+		metric.WithFloat64Callback(reporter.processCpuTimeFunc),
 	)
 	if err != nil {
 		return nil, fmt.Errorf("can't create process time metric: %w", err)
 	}
-	reporter.processMemoryPhysical, err = reporter.meter.Float64ObservableCounter(
+
+	reporter.processMemoryPhysical, err = reporter.meter.Float64ObservableGauge(
```

Comment:
Is changing this metric type a breaking change?

Reply:
it was previously incorrect, but also no one is using this image yet other than me pretty much because we don't have a way to deploy this yet.